content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -377,9 +377,10 @@ metadata:
  name: productpage-viewer
  namespace: default
 spec:
- selector:
-   matchLabels:
-     istio.io/gateway-name: bookinfo-productpage
+  targetRef:
+    kind: Gateway
+    group: gateway.networking.k8s.io
+    name: bookinfo-productpage
```

Comment:
This waypoint targetRef is the only one that doesn't seem to work. Is it not yet supported for ambient?

cc @howardjohn @keithmattix

Reply:
It is supported for waypoints. Based on the test [output](https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio.io/14157/doc.test.profile-none_istio.io/1724520189589983232/artifacts/tests-setup-profile-none-486905/TestDocs/ops/ambient/getting-started/test.sh/test.sh/_test_context/test.sh_output.txt), it looks like the issue is some YAML parsing? 

> error: error parsing STDIN: error converting YAML to JSON: yaml: line 10: did not find expected key

Do you have a tab character somewhere in the YAML?

Reply:
Does this maybe have something to do with it? https://github.com/istio/istio/pull/46560#discussion_r1305894118

Reply:
> Does this maybe have something to do with it? istio/istio#46560 (comment)

I don't believe so; my understanding is that waypoints don't respond to WDS. 

It doesn't look like we have authz policy integration tests for Ambient/waypoints. From a YAML perspective though, the codepaths for regular k8s gateways vs waypoints are the exact same. I checked out the branch and tried to run the `kubectl apply` on a non-Istio cluster and I got the exact same yaml parsing error. In my editor at least, the `action` and `rules` blocks aren't appropriately indented

Reply:
Good catch! I added the targetRef with the proper indentation, but that didn't align with the following incorrectly indented YAML. Thanks!